parser/parser.go

```diff
@@ -542,6 +473,217 @@ SpecLoop:
+	if imp, obj := pkgObj(info, ce.Fun); imp == cronImportPath && obj == "NewJob" {
+		if len(ce.Args) != 2 {
+			p.errf(ce.Pos(), "cron.NewJob must be called as (id string, cfg cron.JobConfig)")
+			return nil
+		}
+
+		cj := &est.CronJob{}
+		if bl, ok := ce.Args[0].(*ast.BasicLit); ok && bl.Kind == token.STRING {
+			cronJobID, _ := strconv.Unquote(bl.Value)
+			if cronJobID == "" {
+				p.errf(ce.Pos(), "cron.NewJob: id argument must be a non-empty string literal")
+				return nil
+			}
+			err := dnsname.DNS1035Label(cronJobID)
+			if err != nil {
+				p.errf(ce.Pos(), "cron.NewJob: id must consist of lower case alphanumeric characters"+
+					" or '-',\n// start with an alphabetic character, and end with an alphanumeric character ")
+				return nil
+			}
+			cj.ID = cronJobID
+			cj.Name = cronJobID // Set ID as the default name
+		} else {
+			p.errf(ce.Pos(), "cron.NewJob must be called with a string literal as its first argument")
+			return nil
+		}
+
+		if cl, ok := ce.Args[1].(*ast.CompositeLit); ok {
+			if imp, obj := pkgObj(info, cl.Type); imp == cronImportPath && obj == "JobConfig" {
+				hasSchedule := false
+				for _, e := range cl.Elts {
+					kv := e.(*ast.KeyValueExpr)
+					key, ok := kv.Key.(*ast.Ident)
+					if !ok {
+						p.errf(kv.Pos(), "cron.JobConfig key must be an identifier")
+						return nil
+					}
+					switch key.Name {
+					case "Name":
+						if v, ok := kv.Value.(*ast.BasicLit); ok && v.Kind == token.STRING {
+							parsed, _ := strconv.Unquote(v.Value)
+							cj.Name = parsed
+						} else {
+							p.errf(v.Pos(), "cron.JobConfig.Name must be a string literal")
+							return nil
+						}
+					case "Every":
+						if hasSchedule {
+							p.errf(kv.Pos(), "cron.JobConfig.Every: cron execution schedule was already defined using the Schedule field, at least one must be set but not both")
+							return nil
+						}
+						if dur, ok := p.parseDurationLiteral(info, kv.Value); ok {
+							// We only support intervals that are a positive integer number of minutes.
+							if rem := dur % minute; rem != 0 {
+								p.errf(kv.Value.Pos(), "cron.JobConfig.Every: must be an integer number of minutes, got %d", dur)
+								return nil
+							}
+
+							minutes := dur / minute
+							if minutes < 1 {
+								p.errf(kv.Value.Pos(), "cron.JobConfig.Every: duration must be one minute or greater, got %d", minutes)
+								return nil
+							} else if minutes > 24*60 {
+								p.errf(kv.Value.Pos(), "cron.JobConfig.Every: duration must not be greater than 1440 minutes (1 day), got %d", minutes)
+							} else if suggestion, ok := p.isCronIntervalAllowed(int(minutes)); !ok {
+								suggestionStr := p.formatMinutes(suggestion)
+								minutesStr := p.formatMinutes(int(minutes))
+								p.errf(kv.Value.Pos(), "cron.JobConfig.Every: 24 hour time range (from 00:00 to 23:59) needs to be evenly divided by the interval value (%s), try setting it to (%s)", minutesStr, suggestionStr)
```

Comment:
It would be helpful to provide the suggestion in the correct syntax instead of just "1 hour 30 mins" or whatever.